QUARKUS-713.md

```diff
@@ -0,0 +1,46 @@
+# QUARKUS-713 Tech preview support for the OpenShift Client
+
+This document describes a high-level testing overview of [QUARKUS-713 Tech preview support for the OpenShift Client](https://issues.redhat.com/browse/QUARKUS-713).
+
+The goal of QUARKUS-713 is to provide the users with tech preview support of utility that will expose an OpenShift
+client. This utility will make it possible for the users to develop their kubectl plugins or OpenShift operators.
+
+## Scope of the testing
+### Smoke testing
+[`OpenShiftClientTest`](https://github.com/quarkusio/quarkus/blob/master/integration-tests/openshift-client/src/test/java/io/quarkus/it/openshift/client/OpenShiftClientTest.java) 
+was implemented as part of feature development. This unit test verifies the basic functional assertions in a mock 
+environment.
+
+This test is executed upstream as a part of Quarkus test suite.
+
+### Component testing
+The feature itself consists of exposing fabric8 OpenShift client to the Quarkus application conveniently.
+
+fabric8 itself has an [upstream CI](https://github.com/fabric8io/kubernetes-client/actions) where the functionality of 
+all the client libraries is verified.
+
+### Integration testing
+An integration test verifying correctness of the feature on OpenShift clusters will be implemented in 
+[quarkus-qe/quarkus-openshift-test-suite](https://github.com/quarkus-qe/quarkus-openshift-test-suite).
+
+The test will consist of a simple operator implemented with the new feature. The operator will watch `configMap` 
+resources in the test namespace and generate a deployment.
+
+### Platform testing
+The tests will be executed on the following platforms:
+
+* OCP 3.11
+* Latest LTS release of OpenShift 4
+* Latest rolling release of OpenShift 4
+
+## Getting familiar with the feature
+Getting introduced to OpenShift operator development and upstream documentation of the Quarkus feature was used as a
+starting point to develop the initial ideas and implementation of simple operators.
+
+## Contacts
+* Tester: Michal Jurč <mjurc@redhat.com>
+
+## References
+* Feature issue: [QUARKUS-713 Tech preview support for the OpenShift Client](https://issues.redhat.com/browse/QUARKUS-713).
+* Upstream feature issue: [Implement OpenshiftClient on Kubernetes Client Extension #3200](https://github.com/quarkusio/quarkus/issues/3200)
+* Upstream documentation: [Quarkus - Kubernetes Client # OpenShift client](https://quarkus.io/guides/kubernetes-client#openshift-client)
```

Comment:
Get rid of this chapter.
Place JIRA link and GH link on the top. Upstream doc can be added as a hyperlink inside "Getting familiar with the feature"

Reply:
I don't particularly like this suggestion, I mean, how are references hurting anything? This is convenient - all the existing stuff is in a single, stylistically expected place. I don't think a document should start with references, either.

Reply:
I have no strong feeling about having summary of the links in "References"

JIRA and GH link are (now?) at the top too, so I'm fine with current format